src/content/platform-overview/app-platform/index.md

```diff
@@ -90,29 +88,54 @@ together to enable the features of the Giant Swarm App Platform:
-
-By default you will have the Giant Swarm Catalog and the Giant Swarm Playground installed
-on your management clusters.
-
-#### The Giant Swarm Catalog
+This catalog contains our stable, fully managed apps, with SLA (e.g. the NGINX Ingress Controller).
 
-This catalog contains our stable, fully managed apps, with SLA (e.g. the NGINX Ingress Controller). It also contain apps that we are developing towards that level of commitment (e.g. Kong, EFK).
-
-Maturity levels of apps in this catalog are expressed through semantic versioning as follows:
+The maturity levels of apps in this catalog are expressed through semantic versioning as follows:
 
 - Version with `-alpha` or `-beta` suffix - the application is only at a basic maturity level. There is no stable release. It is supported on a best effort basis,
 - Version with `-rc*` suffix - the application is at a preview maturity level. This allows customers to preview a new release of an application and evaluate new features. It is supported on a best effort basis.
 - version >= `v1.0.0` with no suffix - the specified version of the application is at a stable maturity level. It is available to our customers as a managed offering with support and SLA.
 
-#### The Giant Swarm Playground
 
-This is our go-to place to create and try out things. Mainly, this contains apps that we have added in order help you with a certain issue. Additionally, you will find some apps that we created for non-commercial purposes (e.g. for a blog post or a workshop).
+### What is a Managed App
+
+A _Managed App_ is consider an app in our Giant Swarm Catalog that provides:
+
+1. Safe and tested deployment
+
+We make sure the Helm chart works, either sanitizing the upstream forked applications or creating good defaults for our maintained ones. We have [a common way of building apps](https://github.com/giantswarm/app-build-suite) and a [testing framework](https://github.com/giantswarm/app-test-suite) which ensures the application is deployable and works as expected. Security and upgradability are checked too during the integration process.
+
+1. Monitoring
+
+Giant Swarm makes sure all the main components of the app are running and that the app is working right. At the same time, we monitor and alert on necessary metrics to ensure our SLAs.
+
+In case of an alert, we perform an RCA (root cause analysis) to understand if it is a Giant Swarm or customer-inflicted issue that broke the application.
+
+__Note__: Usually, we don’t fix bugs upstream when it involves code changes. That said, we try to get find a workaround again and submit a ticket to the upstream project. In some cases, we have fixed the bug ourselves where it’s necessary and possible for us, and provide that to the upstream project. This might result in running a non-upstream patch version from Giant Swarm until upstream merges our patch. Customers can, in general, expect the same level of service for the Managed Optional App as they get with the CoreDNS.
```

Comment:
```suggestion
__Note__: We generally do not fix bugs upstream when they involve significant code changes. That said, we always try to find a workaround or the root cause of the issue and submit a ticket to the upstream project. In some cases, we fix the bug ourselves where it’s necessary and possible for us, and provide the fix to the upstream project. This might result in temporarily running a non-upstream patch version from Giant Swarm until upstream merges our patch. Customers can, in general, expect the same level of service for a Managed Optional App as they get with "default" apps such as CoreDNS.
```